openquake/qa_tests_data/event_based/case_1/expected/gmf-65536.csv

```diff
@@ -1,3 +1,2 @@
-# smlt_path=b1, gsimlt_path=b1, investigation_time=1.0
-lon,lat,PGA
-0.00000,0.00000,9.349059E-02
+sid,eid,imti,gmv
```

Comment:
Please check if changing the format of this csv file we are breaking anything in the HMTK